scripts/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Lets not move scripts to a workspace - it's not a packageable/runnable thing like the packages/playgrounds. These are scripts run from the top level of the repo (via `node scripts/...`) so their deps make sense in the root `package.json`.